RevenueCatUI/CustomerCenter/ButtonStyles.swift

```diff
@@ -37,24 +37,56 @@ struct ManageSubscriptionsButtonStyle: PrimitiveButtonStyle {
         Button(action: { configuration.trigger() }, label: {
             configuration.label.frame(maxWidth: .infinity)
         })
+        .font(.body.weight(.medium))
         .buttonStyle(.borderedProminent)
         .controlSize(.large)
+        .buttonBorderShape(.roundedRectangle(radius: 16))
         .applyIf(background != nil, apply: { $0.tint(background) })
         .applyIf(textColor != nil, apply: { $0.foregroundColor(textColor) })
     }
 
 }
 
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+struct SubtleButtonStyle: PrimitiveButtonStyle {
+
+    @Environment(\.appearance) private var appearance: CustomerCenterConfigData.Appearance
+    @Environment(\.colorScheme) private var colorScheme
+
+    func makeBody(configuration: PrimitiveButtonStyleConfiguration) -> some View {
+        let background = color(from: appearance.buttonBackgroundColor, for: colorScheme)
+
+        Button(action: { configuration.trigger() }, label: {
+            configuration.label.frame(maxWidth: .infinity)
+        })
+        .font(.body.weight(.medium))
+        .controlSize(.large)
+        .padding(.vertical)
+        .applyIf(background != nil, apply: { $0.foregroundColor(background) })
```

Comment:
this isn't using the button text color, because the subtle style is inverted (transparent background, with accented text instead of accented background and customizable text). since I think the intention is for the buttons to be displayed alongside each other with a similar theme, I thought this made the most sense